pytorch_pfn_extras/tou/export_testcase.py

```diff
@@ -0,0 +1,219 @@
+
+
+def _export_meta(model, out_dir, strip_large_tensor_data):
+    ret = {
+        'generated_at': datetime.datetime.now().isoformat(),
+        'output_directory': out_dir,
+        'exporter': 'torch-onnx-utils',
+        'strip_large_tensor_data': strip_large_tensor_data,
+    }
+    try:
+        git_status = subprocess.Popen(['git', 'status'],
+                                      stdout=subprocess.PIPE,
+                                      stderr=subprocess.PIPE)
+        git_status.communicate()
+        if git_status.returncode == os.EX_OK:
+            def strip_cmd(cmd):
+                return os.popen(cmd).read().strip()
+            ret['git'] = {
+                'branch': strip_cmd('git rev-parse --abbrev-ref HEAD'),
+                'commit': strip_cmd('git rev-parse HEAD'),
+                'remote': strip_cmd('git ls-remote --get-url origin'),
+                'commit_date': strip_cmd('git show -s --format=%ci HEAD'),
+            }
+    except FileNotFoundError:
+        pass
+
+    return ret
+
+
+def _export_util(model, args, f, **kwargs):
+    """Wrap operator type to export
+
+    Copied from torch.onnx.utils.export
+    """
+    aten = kwargs.get('aten', False)
+    export_raw_ir = kwargs.get('export_raw_ir', False)
+    operator_export_type = kwargs.get('operator_export_type', None)
+    if aten or export_raw_ir:
+        assert operator_export_type is None
+        assert aten ^ export_raw_ir
+        operator_export_type = OperatorExportTypes.ATEN if\
+            aten else OperatorExportTypes.RAW
+    elif operator_export_type is None:
+        if torch.onnx.PYTORCH_ONNX_CAFFE2_BUNDLE:
+            operator_export_type = OperatorExportTypes.ONNX_ATEN_FALLBACK
+        else:
+            operator_export_type = OperatorExportTypes.ONNX
+
+    return _export(model, args, f, _retain_param_name=True, **kwargs)
+
+
+def is_large_tensor(tensor, threshold):
+    size = reduce(operator.mul, tensor.dims, 1)
+    return size > threshold
+
+
+def _strip_raw_data(tensor):
+    arr = onnx.numpy_helper.to_array(tensor)
+    meta_dict = {}
+    meta_dict['type'] = "stripped"
+    meta_dict['average'] = float(numpy.average(arr))
+    meta_dict['variance'] = float(numpy.var(arr))
+    onnx.external_data_helper.set_external_data(
+            tensor,
+            location=json.dumps(meta_dict),
+            length=len(tensor.raw_data))
+    tensor.data_location = onnx.TensorProto.EXTERNAL
+    tensor.ClearField('raw_data')
+    return tensor
+
+
+def _strip_large_initializer_raw_data(onnx_path, large_tensor_threshold):
+    model = onnx.load(onnx_path)
+    for init in model.graph.initializer:
+        if is_large_tensor(init, large_tensor_threshold):
+            _strip_raw_data(init)
+
+    # Overwrite onnx file
+    with open(onnx_path, "wb") as model_file:
+        model_file.write(model.SerializeToString())
+
+
+def export_testcase(
+        model, args, out_dir, output_grad=False, metadata=True,
```

Comment:
Maybe this is better later enhancement?
```suggestion
        model, args, out_dir, *, output_grad=False, metadata=True,
```